src/components/common/Header.js

```diff
@@ -39,13 +40,17 @@ function MainHeader(props) {
     return () => window.removeEventListener('scroll', handleScroll);
   }, [oldScroll, showHeader, handleScroll]);
 
-  const { searchDocs } = props;
+  const { getDocs, searchDocs } = props;
   const {
     authService: { logout },
     authState,
   } = useOktaAuth();
   const { pathname } = useLocation();
 
+  const bookmarksButton = () => {
+    getDocs(authState);
```

Comment:
Really like how you got this to work to trigger auth state to get everything updated. It's nice that you don't have to go to the page and then have to re-refresh it after arriving which is a problem I've seen with state before.